cmd/snap/cmd_unset.go

```diff
@@ -39,6 +39,13 @@ Nested values may be removed via a dotted path:
 	$ snap unset snap-name user.name
 `)
 
+var longAspectUnsetHelp = i18n.G(`
+With the aspects experimental feature enabled, if the first argument passed
```

Comment:
Prefixing the sentence with `With the aspects experimental feature enabled...` reads a little weird. I think that it might be better if it were converted to a standalone sentence that explains that this only works if the aspects experimental feature is enabled.

Reply:
Since we only show this if the flag is already enabled, I removed that sentence to simplify things a bit. Thanks